cli/sql_util.go

```diff
@@ -174,9 +173,18 @@ func formatVal(val interface{}) string {
 	case nil:
 		return "NULL"
 	case []byte:
-		if !utf8.Valid(t) {
-			// Ensure that protobufs containing non-UTF8 binary data print escaped.
-			return fmt.Sprintf("%q", t)
+		// Ensure that buffers containing non-printable ASCII characters print escaped.
+		printable := true
+		for _, b := range t {
+			// Printable ASCII characters are between 0x20 and 0x7E.
+			if b < 0x20 || b > 0x7E {
```

Comment:
Oh, there is also [`unicode.IsControl`](https://godoc.org/unicode#IsControl) which might be useful.